packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -96,6 +98,10 @@ export class MgtFileList extends MgtTemplatedTaskComponent implements CardSectio
     return strings;
   }
 
+  // files from the person card component
+  @state()
+  private _files: DriveItem[];
```

Comment:
```suggestion
  private personCardFiles: DriveItem[];
```
Let us use a more descriptive name instead.